src/pages/patch/patchTabs/tasks/TasksTable.tsx

```diff
@@ -62,6 +64,65 @@ const getSortDirFromOrder = (order: "ascend" | "descend") =>
 
 const rowKey = ({ id }: { id: string }): string => id;
 
+const mapTaskStatusToBadgeVariant = {
+  [TaskStatus.Inactive]: Variant.LightGray,
+  [TaskStatus.Unstarted]: Variant.LightGray,
+  [TaskStatus.Undispatched]: Variant.LightGray,
+  [TaskStatus.Started]: Variant.Yellow,
+  [TaskStatus.Dispatched]: Variant.LightGray,
```

Comment:
Dispatched should be the same yellow that started is

Reply:
thanks!!